doc/blog-post/2017-07-xx-1.3.0-announcement.md

```diff
@@ -0,0 +1,160 @@
+
+```js
+var dateWithTimeZoneFormatter = Globalize.dateFormatter({
+  datetime: "full",
+  timeZone: "America/Los_Angeles"
+});
+```
+
+The final size (for production) of this code will be:
+
+| filename                                 | minified+gzipped size |
+| ---------------------------------------- | --------------------- |
+| i18n/en.js (includes CLDR and IANA data) | 1.7KB                 |
+| core, number, and date globalize runtime lib + zoned-date-time | 7.0KB                 |
+
+See globalize [compiler example](https://github.com/globalizejs/globalize/tree/master/examples/globalize-compiler) or [app-npm-webpack example](https://github.com/globalizejs/globalize/tree/master/examples/app-npm-webpack) for details.
+
+## Format Date To Parts
+
+> This change was contributed by Reza Payami @rpayami (PayPal) and Rafael Xavier @rxaviers in #697 and #700.
+
+Modern user interfaces often need to manipulate the date format output, which is impossible via the existing format function that returns an opaque string and makes any attempt to do it subject to break internationalization. [Ecma-402](https://github.com/tc39/ecma402/) has recently added [`Intl.DateTimeFormat.prototype.formatToParts`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat/formatToParts) to fulfill that purpose, which at the time of this writing is at stage 4 and is implemented by latest Firefox and Chrome.
+
+In Globalize, we introduced [`.dateToPartsFormatter`](https://github.com/globalizejs/globalize/blob/master/doc/api/date/date-to-parts-formatter.md) and [`.formatDateToParts`](https://github.com/globalizejs/globalize/blob/master/doc/api/date/date-to-parts-formatter.md).
+
+```js
+Globalize.locale( "en" );
+Globalize.formatDateToParts(new Date(2010, 10, 30));
+// > [
+//   { "type": "month", "value": "11" },
+//   { "type": "literal", "value": "/" },
+//   { "type": "day", "value": "30" },
+//   { "type": "literal", "value": "/" },
+//   { "type": "year", "value": "2010" }
+// ]
+```
+
+The information is made available separately and it can be formatted and concatenated again in a customized way. For example by using [`Array.prototype.map()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map), [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions), a [switch statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch), [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), and [`Array.prototype.reduce()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce).
```

Comment:
Looks like its not using reduce anymore, and the use of the switch statement is pretty odd - an if would do just fine.

I don't think `.map` needs an MDN link.